deploy/helm/node-feature-discovery-operator/templates/service.yaml

```
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  labels:
+    control-plane: controller-manager
+    {{- include "node-feature-discovery-operator.labels" . | nindent 4 }}
+  name: nfd-controller-manager-metrics-service
```

Fullname here, too(?)

```suggestion
  name: {{ include "node-feature-discovery-operator.fullname" . }}-controller-manager-metrics
```